rllib/utils/exploration/tests/test_explorations.py

```diff
@@ -28,6 +28,9 @@ def do_test_explorations(config, dummy_obs, prev_a=None, expected_mean_action=No
         for exploration in [None, "Random"]:
             local_config = config.copy()
             if exploration == "Random":
+                if local_config._enable_rl_module_api:
+                    # TODO(Artur): Support Random exploration with RL Modules.
+                    continue
                 local_config.exploration(exploration_config={"type": "Random"})
```

Comment:
This will fail because it simply does nothing today.
I don't think that we need to test an alternative right now, since a random behavior can be defined in the exploration function.